contracts/programs/bubblegum/src/lib.rs

```diff
@@ -483,8 +468,15 @@ pub mod bubblegum {
             data_hash,
             creator_hash,
         );
-        let new_leaf =
-            LeafSchema::new_v0(asset_id, owner, new_delegate, nonce, data_hash, creator_hash);
+        let new_leaf = LeafSchema::new_v0(
+            asset_id,
+            owner,
+            new_delegate,
+            nonce,
+            data_hash,
+            creator_hash,
+        );
+        wrap_event(new_leaf.try_to_vec()?, &ctx.accounts.candy_wrapper)?;
```

Comment:
Does `wrap_event` need more context or is the instruction tree sufficient to interpret these events?

Reply:
Instruction tree is sufficient to interpret events, but assumes that indexer is always able to read the `bubblegum` instruction logic